devices/develco/zhemi101_external_meter_interface.json

```diff
@@ -0,0 +1,223 @@
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion",
+          "refresh.interval": 86400,
+          "read": {
+            "at": "0x8000",
+            "cl": "0x0000",
+            "ep": 2,
+            "fn": "zcl",
+            "mf": "0x1015"
+          },
+          "parse": {
+            "at": "0x8000",
+            "cl": "0x0000",
+            "ep": 2,
+            "fn": "zcl",
+            "mf": "0x1015",
+            "script": "develco_firmware.js"
+          }
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/interfacemode",
+          "parse": {
+            "at": "0x0302",
+            "cl": "0x0702",
+            "ep": 2,
+            "eval": "let v = [0x0000, 0x0001, 0x0002, 0x0100, 0x0101, 0x0102, 0x0103, 0x0104].indexOf(Attr.val); Item.val = v != -1 ? v + 1 : 0;",
+            "fn": "zcl",
+            "mf": "0x1015"
+          },
+          "read": {
+            "at": "0x0302",
+            "cl": "0x0702",
+            "ep": 2,
+            "fn": "zcl",
+            "mf": "0x1015"
+          },
+          "write": {
+            "at": "0x0302",
+            "cl": "0x0702",
+            "dt": "0x31",
+            "ep": 2,
+            "eval": "[0x0000, 0x0001, 0x0002, 0x0100, 0x0101, 0x0102, 0x0103, 0x0104][Item.val - 1];",
+            "fn": "zcl",
+            "mf": "0x1015"
+          },
+          "values": [
+            [1, "Pulse counting electricity - Unit kWh"],
+            [2, "Pulse counting gas - Unit m3"],
+            [3, "Pulse counting water - Unit m3"],
+            [4, "Kamstrup KMP protocol"],
+            [5, "Linky protocol"],
+            [6, "DLMS-COSEM - IEC62056-21 mod A"],
+            [7, "P1 dutch standard – DSMR 2.3 version"],
+            [8, "P1 dutch standard – DSMR 4.0 version"]
+          ]
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/pulseconfiguration",
+          "parse": {
+            "at": "0x0300",
+            "cl": "0x0702",
+            "ep": 2,
+            "eval": "Item.val = Attr.val",
+            "mf": "0x1015"
+          },
+          "read": {
+            "at": "0x0300",
+            "cl": "0x0702",
+            "ep": 2,
+            "fn": "zcl",
+            "mf": "0x1015"
+          },
+          "write": {
+            "at": "0x0300",
+            "cl": "0x0702",
+            "dt": "0x21",
+            "ep": 2,
+            "eval": "Item.val;",
+            "fn": "zcl",
+            "mf": "0x1015"
+          }
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/consumption",
+          "refresh.interval": 360,
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0702",
+            "ep": 2,
+            "eval": "Item.val = Attr.val / 1000",
```

Comment:
Hi!
After I updated to the new Deconz version in Home Assistant I noticed that the values from my sensor were suddenly a factor 1000 too small. I bought the EU version of the sensor and live in EU where the standard is 1000 pulses/kWh, which the sensor assumes. I managed to fix it by manually editing the DDF and removing the scaling, but I would suggest not scaling the value in the default DDF, since this will most likely break the values for many users.

Reply:
```suggestion
            "eval": "Item.val = Attr.val",
```